packages/cw0/src/event.rs

```diff
@@ -0,0 +1,5 @@
+use cosmwasm_std::Attribute;
+
+pub trait Event {
```

Comment:
Can you add rustdoc on this?

I am sure I will see how it is used a few files down, but if someone comes to it for the first time, nice to explain.

Reply:
done